app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The same message 'All friends should be vaccinated' is used for both `NotVaccinatedError` and `OutdatedVaccineError`. Consider providing distinct messages for each exception to give more specific feedback about the issue.